docs/generate/generate.md

```diff
@@ -8,26 +8,26 @@ The API classes in this package provide a simple interface to read one or an ent
 
 For a single RAML file:
 ```javascript
-import { Api } from "@commerce-apps/raml-toolkit";
+import { generate } from "@commerce-apps/raml-toolkit";
 
-Api.init("path/to/my-spec.raml").then(console.log);
+generate.Api.init("path/to/my-spec.raml").then(console.log);
```

Comment:
Is it just me, or does this feel more clunky and doesn't seem to flow as well?

Reply:
Doesn't matter, #116 gets rid of it.